dataprofiler/profilers/helpers/report_helpers.py

```diff
@@ -100,7 +118,50 @@ def _prepare_report(report, output_format=None, omit_keys=None):
         if isinstance(value, set):
             value = sorted(list(value))
 
-        if isinstance(value, dict):
```

Comment:
above at `if key in omit_keys:` if we kept original structure, could either utilize regex or for string vs int column name can convert key to `str` for comparison.

Also, i wonder if this should also count any place that has a `*` e.g. `data_stats.*` wouldn't work here would it?